apiary.apib

```diff
@@ -4,7 +4,107 @@ FORMAT: 1A
 
 The API for interacting with IPFS nodes.
 
-# Group add
+# Group add [POST /add{?arg}{&recursive,quiet,silent,progress,trickle,only%2Dhash,wrap%2Dwith%2Ddirectory,hidden,chunker,pin}]
+Add an object to IPFS.
+
+Note that directories are added recursively, to form the ipfs
+MerkleDAG. A smarter partial add with a staging area (like git)
+remains to be implemented.
+
+#### curl
+
+    curl -F "file=@test/" -r http://localhost:5001/api/v0/add
```

Comment:
What would you suggest we do? Shouldn't postman show us some of the output? 

How do I get this output, then?